src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -35,9 +35,8 @@ internal static partial class CFamilyHelper
     {
         public const string CPP_LANGUAGE_KEY = "cpp";
         public const string C_LANGUAGE_KEY = "c";
-        internal const string PchFilePathSuffix = "SonarLintPCH.preamble";
 
-        internal static readonly string PchFilePath = Path.Combine(Path.GetTempPath(), PchFilePathSuffix);
+        internal static readonly string PchFilePath = Path.Combine(Path.GetTempPath(), "SonarLintPCH.preamble");
```

Comment:
I think the name should make it clear that it's SonarLint for VS (SL for VS Code and the other SL flavours also create files, and it would help if we could tell them apart easily).

Maybe:
`SonarLintForVS.PCH.preamble`
`SonarLintForVisualStudio.PCH.preamble`
... just as long as it's clearly VS not VSCode / IntelliJ / Eclipse